scm-ui/ui-components/src/repos/changesets/SignatureIcon.tsx

```diff
@@ -0,0 +1,119 @@
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import React, {FC} from "react";
+import {useTranslation} from "react-i18next";
+import {Signature} from "@scm-manager/ui-types";
+import styled from "styled-components";
+import Icon from "../../Icon";
+import {usePopover} from "../../popover";
+import Popover from "../../popover/Popover";
+import classNames from "classnames";
+
+type Props = {
+  signatures: Signature[];
+  className: any;
+};
+
+const StyledIcon = styled(Icon)`
+  width: 1em;
+  height: 1em;
+  vertical-align: middle;
+  border-radius: 0.25em;
+  margin-bottom: 0.2em;
+`;
+
+const StyledDiv = styled.div`
+  > *:not(:last-child) {
+    margin-bottom: 24px; 
+  }
+`;
+
+const SignatureIcon: FC<Props> = ({signatures, className}) => {
+  const [t] = useTranslation("repos");
+  const {popoverProps, triggerProps} = usePopover();
+
+  if (!signatures.length) {
+    return null;
+  }
+
+  const getColor = (signatures: Signature[]) => {
+    const invalid = signatures.some(sig => sig.status === "INVALID");
+    if (invalid) {
+      return "danger";
+    }
+    const verified = signatures.some(sig => sig.status === "VERIFIED");
+    if (verified) {
+      return "success";
+    }
+    return undefined;
+  };
+
+  const createSignatureBlock = (signature: Signature) => {
+    let status;
+    if (signature.status === "VERIFIED") {
+      status = t("changeset.signatureVerified");
+    } else if (signature.status === "INVALID") {
+      status = t("changeset.signatureInvalid");
+    } else {
+      status = t("changeset.signatureNotVerified");
+    }
+
+    if (signature.status === "NOT_FOUND") {
+      return <p>
+        <div>{t("changeset.keyId")}: {signature.keyId}</div>
+        <div>{t("changeset.signatureStatus")}: {status}</div>
+      </p>;
+    }
+
+    return <p>
+      <div>{t("changeset.keyId")}: {
+        signature._links?.rawKey ? <a href={signature._links.rawKey.href}>{signature.keyId}</a> : signature.keyId
+      }</div>
+      <div>{t("changeset.signatureStatus")}: <span className={classNames(`has-text-${getColor([signature])}`)}>{status}</span></div>
+      <div>{t("changeset.keyOwner")}: {signature.owner || t("changeset.noOwner")}</div>
+      {signature.contacts && signature.contacts.length > 0 && <>
+        <div>{t("changeset.keyContacts")}:</div>
+        {signature.contacts && signature.contacts.map(contact =>
+          <div>- {contact.name}{contact.mail && ` <${contact.mail}>`}</div>)}
+      </>}
+    </p>;
+  };
+
+  const signatureElements = signatures.map(signature => createSignatureBlock(signature));
+
+  return (
+    <>
+      <Popover title={<h1>{t("changeset.signatures")}</h1>} width={500} {...popoverProps}>
```

Comment:
`h1` has no style by default. It should be bold at least.

Reply:
```suggestion
      <Popover title={<h1 className="has-text-weight-bold">{t("changeset.signatures")}</h1>} width={500} {...popoverProps}>
```

https://bulma.io/documentation/helpers/typography-helpers/#text-weight

Reply:
```suggestion
      <Popover title={<h1 className="has-text-weight-bold is-size-5">{t("changeset.signatures")}</h1>} width={500} {...popoverProps}>
```

https://bulma.io/documentation/helpers/typography-helpers/#text-weight

We have to check style in all places, because the popover does not use a portal and could potentially inherit styles from its parent.